src/components/MediaSettings/MediaSettings.vue

```diff
@@ -310,13 +306,15 @@ export default {
 			stopDevices,
 			virtualBackground,
 			model: localMediaModel,
+			tabs,
+			isVirtualBackgroundAvailable,
 		}
 	},
 
 	data() {
 		return {
 			modal: false,
-			tabContent: 'none',
+			tabContent: undefined,
```

Comment:
We got a browser storage check in showModal() method to automatically open devices tab. Doesn't seem to work now:

```js
	if (!BrowserStorage.getItem('audioInputDevicePreferred') || !BrowserStorage.getItem('videoInputDevicePreferred')) {
		this.tabContent = 'devices'
	}
```

Reply:
Fixed